server/admin.proto

```diff
@@ -225,6 +225,15 @@ message GetUIDataResponse {
   map<string, Value> key_values = 1;
 }
 
+// ClusterRequest requests metadata for the cluster.
+message ClusterRequest {
+}
+
+// ClusterResponse contains metadata for the cluster.
+message ClusterResponse {
+  string cluster_id = 1 [(gogoproto.customname) = "ClusterID"];
```

Comment:
hm, could you customtype uuid.UUID as is done elsewhere? you'd save a string allocation, at least.